client/src/actors/registry.rs

```diff
@@ -0,0 +1,229 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! Registry Actor
+//!
+//! The registry actor keeps record of all client actors
+//! bound to a unique `client_id`. [`Client`] actors can
```

Comment:
```suggestion
//! bound to a unique `client_id`. [`SecureClient`] actors can
```
Check & fix intra doc links:
```sh 
$ RUSTDOCFLAGS="--deny broken_intra_doc_links" cargo doc --verbose --workspace --no-deps --document-private-items
```